llama_index/embeddings/cohereai.py

```diff
@@ -111,7 +111,7 @@ def __init__(
             raise ValueError(f"truncate must be one of {VALID_TRUNCATE_OPTIONS}")
 
         super().__init__(
-            cohere_client=cohere.Client(cohere_api_key),
+            cohere_client=cohere.Client(cohere_api_key, client_name="llama_index"),
```

Comment:
it is possible users who are using our pkg calling cohere api. the source in this case are not from llama_index.
you can add client_name as a parameter but let user set it.